File: add_n_num/add_n_num.rb
arr=[]
puts "Enter how many elements you want to add"
a=gets.to_i
puts "Enter the elements that you want to add"
for i in 0...a
   arr[i]=gets.to_i
end
sum=0
arr.each { |a| sum+=a }
puts "sum of elements is #{sum}"

File: add_n_num/add_n_num.c
#include <stdio.h>
 
int main()
{
   int n, sum = 0, i, value;
 
   printf("Enter the number of integers you want to add\n");
   scanf("%d", &n);
 
   printf("Enter %d integers\n",n);
 
   for (i = 1; i <= n; i++)
   {
      scanf("%d",&value);
      sum = sum + value;
   }
   printf("Sum of entered integers = %d\n",sum);
 
   return 0;
}
